### PR TITLE
improve usage of appraisal

### PR DESCRIPTION
**Pull request by skatkov**
This is a clean-up for this gem and testing related improvements.

### CODE REVIEW

**Review by julik — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+      - uses: actions/checkout@v3
 
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
+      - uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
           ruby-version: ${{ matrix.ruby }}
 
-      - name: Setup Appraisal
```
Comment (skatkov) [inline .github/workflows/test.yml:21]: Have to revert this change
